guides/hosting/installation-updates/cluster-setup.md

```diff
@@ -4,6 +4,20 @@ The setup of high-scaling systems differs from a normal installation of Shopware
 
 This guide contains information for everyone who intends to start with such a project.
 
+## Shopware configuration
+
+{% hint style="info" %}
+This configuration is available starting with Shopware version 6.5.6.0
+{% endhint %}
```

Comment:
we should mention what it does.

Reply:
I added some explanations

Reply:
I still don't get what it means. Like a cache clear of symfony cache files?

Reply:
Yes that is the first specific use case: https://github.com/shopware/platform/commit/1508ddc7d3d051de0a99f348d6abeeae00e534c1

but there may come more cases we might improve, that why i tried to keep it more generic ;) 

But i can add that as an example